### PR TITLE
Gif support

### PR DESCRIPTION
**Pull request by davigonz**
Implements #702

### CODE REVIEW

**Comment by davigonz** [comment]
@jesmrec I've replaced the library to make zoom in the pictures with a new one (Photoview) and use also a new library (Glide) to reproduce the gifs and show the images, I tell you in case you need to include more tests.

**Comment by jesmrec** [comment]
Approved

File: build.gradle [hunk 1]
```diff
@@ -40,7 +44,6 @@ configurations.all {
 
 dependencies {
     /// dependencies for app building
-    compile project(':touchImageView')
```
Comment (hannesa2) [inline build.gradle]: Btw, I was annoyed by no maintenance of this lib. But after going my own way and having a vital fork, Mike Ortiz made me to a maintainer and as you can see, there is now some life there https://github.com/MikeOrtiz/TouchImageView/graphs/contributors,
but anyway, if you are happy with `com.github.chrisbanes:PhotoView` it's perfect !
@michaelstingl @davigonz

Comment (davigonz) [reply build.gradle]: We decided to use PhotoView because, as you can see in the files changed tab, we have pretty less code.

Comment (hannesa2) [reply build.gradle]: It's a library, and in library it's one **one** relevant file.

**Pull request merged**